solutions/USACO dec15-high-card-low-card-gold.mdx

```diff
@@ -11,6 +11,86 @@ author: Qi Wang
 
 <LanguageSection>
 
+<CPPSection>
+
+```cpp
+#include <bits/stdc++.h>
+using namespace std;
+
+typedef  long long ll;
+typedef long double ld;
+
+#define sz(x) (int)(x).size()
+#define endl "\n"
+#define ioss ios::sync_with_stdio(false);cin.tie(0);
```

Comment:
Please use no more macros than what is allowed here:
https://usaco.guide/general/contributing/#c-template